src/screens/CreatePost/index.js

```diff
@@ -718,7 +718,7 @@ const CreatePost = () => {
       animatedReminder.setValue(0);
     }
   }, [typeUser]);
-
+  console.log({typeUser}, 'heja');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line seems to be a debugging statement that was accidentally left in the code. It's generally not a good practice to leave console logs in production code as it can expose sensitive information and clutter the console output.
```diff
-  console.log({typeUser}, 'heja');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->